pages/tickets.tsx

```diff
@@ -15,6 +16,19 @@ class TicketPage extends React.Component<WithPageMetadataProps> {
     }
     return {}
   }
+
+  componentDidMount() {
```

Comment:
Thinking about this. We should implement `componentWillUnmount` to remove the script to avoid injection multiple times, say if you revisit the page.

Reply:
Or even better, let's use object detection i.e. `if (!tito) { ... }` where `tito` is some sort of object their javascript defines.

Reply:
done